interfaces/lego-boost/README.md

```diff
@@ -0,0 +1,246 @@
+You will see the configuration parameters for the Lego Boost robot as follows:
+
+![Image of server interface](../../resources/img/legoboost_config.png) 
+
+Modify the parameters to match your configuration.
+Make sure the object name is the same one as the object you have created ('legoBoost').
+Finally, set enableMIRConnection to true, so that the software tries to connect to the robot.
+
+Once this is all setup, stop your server. Go to the next section: Testing.
+
+### Testing
+
+At this point, you have everything setup and you should test to make sure that everything works together.
+Follow these steps to ensure that your code runs properly.
+
+* Make sure the computer running the server and the smartphone with the Vuforia Spatial Toolbox application are on the same network.
+* On your terminal, navigate to the folder 'vuforia-spatial-edge-server/'
+* Run the server by typing:
+
+```
+node server.js
+```
+
+* Open the browser and go to:
+```
+localhost:8080
+```
+
+* Go to ManageHardwareInterfaces
+* Your robot hardware interface should be ON.
+* Check that the isRobotConnected parameter is set to ON. This means the connection to the robot was successful
+
+* Open the Vuforia Spatial Toolbox application on the phone, POINTING AT THE FLOOR (or surface where your robot will be moving on). The first time the application sees a surface, it will register this surface as the floor. We are currently in the process of implementing user feedback for this. For now, just make sure that you are pointing at the floor when you turn on the application.
+* On the side menu on the phone, click on the configuration gear
+* Go to 'Found Objects'
+
+Here, you will see a list of Objects that the server is looking for. Your object should be on this list.<br />
+If your object is not on this list, something went wrong. Check out our troubleshooting section.
+
+If your object is on the list, point with the phone at your physical object (aka your robot).
+If your object target works, the main UI of the motion tool should show up on the device.
+
+![Image of VST tool](../../resources/img/legoBoost_tracked.jpg)
+
+Now you can point on the floor and tap at any new position where you want a new checkpoint. You will see how a path gets created as you add more checkpoints on the floor.
+Once your path is finished. Connect a button to the first checkpoint.
+
+![Image of VST tool](../../resources/img/legoBoost_button.jpg)
+
+ When you activate this button, the robot will start following the path.
+ 
+ ![Image of VST tool](../../resources/gifs/boost.gif)
+
+### Authors
+
+* **[Anna Fuste](https://github.com/afustePTC)**
+
+See also the list of [contributors](https://github.com/your/project/contributors) who participated in this project.
+
+### License
+
+This project is licensed under the MPL 2.0 License - see the [LICENSE](../../LICENSE) file for details
```

Comment:
This would be better as a full url so it works properly on the website https://github.com/ptcrealitylab/vuforia-spatial-robotic-addon/blob/master/LICENSE